tests/product/test_server_install.py

```diff
@@ -193,7 +202,7 @@ def test_install_with_java8_home(self):
         expected = installed_all_hosts_output
 
         actual = cmd_output.splitlines()
-        self.assertEqual(sorted(expected), sorted(actual))
+        self.assertRegexpMatchesLineByLine(actual, expected)
```

Comment:
Yea that's right. Internally, assertRegexpMatchesLineByLine sorts expect and actual and does a regexp comparison line by line (rather than just string equality as before). I made this change because I added some regex strings to the expected output, so I couldn't just test on string equality.